catalog/imaging/catalog-video-transformer/src/test/java/ddf/catalog/transformer/video/TestStanag4609ProcessorImpl.java

```diff
@@ -0,0 +1,208 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.transformer.video;
+
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import org.codice.alliance.libs.stanag4609.DecodedKLVMetadataPacket;
+import org.codice.ddf.libs.klv.KlvContext;
+import org.codice.ddf.libs.klv.KlvDataElement;
+import org.codice.ddf.libs.klv.KlvDecodingException;
+import org.codice.ddf.libs.klv.data.Klv;
+import org.codice.ddf.libs.klv.data.set.KlvLocalSet;
+import org.junit.Test;
+
+import ddf.catalog.data.Attribute;
+
+public class TestStanag4609ProcessorImpl {
+
+    private static final KlvHandler EMPTY_HANDLER = new KlvHandler() {
```

Comment:
Instead of creating empty or simple implementations, you can mock `KlvHandler` and use Mockito to verify that certain methods were called.